src/state/order_schema/order_packet.rs

```diff
@@ -644,26 +674,34 @@ impl OrderPacket {
 }
 
 pub fn decode_order_packet(bytes: &[u8]) -> Option<OrderPacket> {
-    let order_packet = match OrderPacket::try_from_slice(bytes) {
-        Ok(order_packet) => order_packet,
+    // The optional fields at the end of the order packet are not required in the raw input data.
+    let additional_fields = &[
+        0_u8, /* last_valid_slot */
+        0_u8, /* last_valid_unix_timestamp_in_seconds */
+        0_u8, /* fail_silently_on_insufficient_funds */
+    ];
```

Comment:
here, are we assuming that any new fields that will get added in the future will get added to all OrderPackets? are we ok with that?

Reply:
and technically shouldn't these be reversed in order?